demos/gesture_recognition_demo/cpp_gapi/README.md

```diff
@@ -0,0 +1,125 @@
+# G-API Gesture Recognition Demo
+
+This demo demonstrates how to run Gesture (e.g. American Sign Language (ASL) gestures) Recognition models using OpenVINO&trade; toolkit.
+
+## How It Works
+
+The demo application expects a gesture recognition model in the Intermediate Representation (IR) format.
+
+As input, the demo application takes:
+
+* a path to a video file or a device node of a webcam specified with a command line argument `--input`
+* a path to a file in JSON format with gesture class names `--class_map`
+
+The demo workflow is the following:
+
+1. The demo application reads video frames one by one, runs person detector that extracts ROI, tracks the ROI of very first person. Additional process is used to prepare the batch of frames with constant framerate.
+2. Batch of frames and extracted ROI are passed to artificial neural network that predicts the gesture.
+3. The app visualizes results of its work as graphical window where following objects are shown:
+    - Input frame with detected ROI.
+    - Last recognized gesture.
+    - Performance characteristics.
+
+> **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with the `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvino.ai/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model.html#general-conversion-parameters).
+
+## Creating a Gallery for gestures window
+
+To show gestures on a additional window, the demo needs a gallery of reference videos. Each video should contain an appropriate name of gesture. You can create the gallery from an arbitrary list of images:
```

Comment:
```suggestion
The gallery of sample videos can be created to show the sample gestures on an additional window:
```

Reply:
Fixed. But I think need to change comment one more time. Script creates list of paths to videos in gallery.

Reply:
ok, please change